OpenRA.Game/Network/Order.cs

```diff
@@ -21,7 +21,8 @@ public enum OrderType : byte
 		SyncHash = 0x65,
 		Disconnect = 0xBF,
 		Handshake = 0xFE,
-		Fields = 0xFF
+		Fields = 0xFF,
+		Ack = 0x10
```

Comment:
This will need a bump in the protocol version.

Reply:
Now that there's a lobby option `UseNewNetcode` it shouldn't, I'd expect this to be backward compatible with existing servers.